include/boost/geometry/strategies/spherical/distance_cross_track_box_box.hpp

```diff
@@ -162,15 +162,15 @@ public :
                     <
                         Strategy, box_point_type1, box_point_type2
                     >::apply(ps_strategy, ps_strategy.get_distance_strategy()
-                               .meridian(lat_min1, lat_max2));
+                               .vertical_or_meridian(lat_min1, lat_max2));
```

Comment:
Right, this function should be implemented in this strategy or some details, not in P/P strategy. It's neither needed in P/P algorithm nor P/S algorithm. It's only needed here in S/B strategy.